pkg/validations/base_test.go

```diff
@@ -109,3 +129,34 @@ func TestRunValidationsIssueCorrection(t *testing.T) {
 		t.Errorf("Deployment test failed %#v: got %d want %d", checkName, metricValue, 0)
 	}
 }
+
+func TestIncompatibleChecksAreDisabled(t *testing.T) {
+	e, err := createEngineWithAllChecks()
+	if err != nil {
+		t.Errorf("Error creating validation engine %v", err)
+	}
+
+	enabledChecks := e.EnabledChecks()
+	if len(enabledChecks) <= 10 {
```

Comment:
This will change as we add checks.  Can we dynamically determine what the expected number of enabled checks is?

Reply:
The current number is actually 37 checks (excluding the disabled checks). '10' was an arbitrary number I chose to make sure we weren't loading an empty list or a list from the other test.

With that said, I'll look at rewriting this piece.

Reply:
Resolved with:
https://github.com/app-sre/deployment-validation-operator/blob/8fe3c3289499cb0c2ec8433920c0b7ccbb7e2551/pkg/validations/base_test.go#L128-L139